pkg/api/types.go

```diff
@@ -410,10 +410,11 @@ func IsInvalidFilename(name string) bool {
 // When the destination is not specified, the source get copied into current
 // working directory in container.
 func (l *VolumeList) Set(value string) error {
+	if len(value) == 0 {
+		return fmt.Errorf("invalid format, must be source:destination")
+	}
 	mount := strings.Split(value, ":")
 	switch len(mount) {
-	case 0:
-		return fmt.Errorf("invalid format, must be source:destination")
```

Comment:
It turns out that `strings.Split("", ":")` returns slice with single element (instead of empty slice).